src/panels/lovelace/hui-editor.ts

```diff
@@ -133,7 +133,7 @@ class LovelaceFullConfigEditor extends LitElement {
         }
 
         .content {
-          height: calc(100vh - 68px);
+          height: calc(100vh - calc(var(--app-header-height) + 4px));
```

Comment:
What is the extra 4px for?

Reply:
Theres a border/spacing on the bottom of the "Raw configuration editor" page of 4 px